docs/rule/no-obsolete-elements.md

```diff
@@ -0,0 +1,66 @@
+## no-obsolete-elements
+
+Some elements are entirely obsolete and must not be used by authors. 
+
+This rule forbids the use of obsolete elements.
+
+### Examples
+
+This rule **forbids** the following:
+
+```hbs
+  <applet></applet>
+  <acronym></acronym>
```

Comment:
There is some odd indentation in this list, can you make them all indented the same way?